[2022-06-27 13:31:41,494] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-03-08T00:00:00+00:00 [queued]>
[2022-06-27 13:31:41,562] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-03-08T00:00:00+00:00 [queued]>
[2022-06-27 13:31:41,569] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:41,572] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:31:41,575] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:41,660] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_B> on 2022-03-08 00:00:00+00:00
[2022-06-27 13:31:41,700] {standard_task_runner.py:52} INFO - Started process 57379 to run task
[2022-06-27 13:31:41,757] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_B', 'scheduled__2022-03-08T00:00:00+00:00', '--job-id', '342', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpfqntoc2n', '--error-file', '/tmp/tmphznj2tso']
[2022-06-27 13:31:41,766] {standard_task_runner.py:80} INFO - Job 342: Subtask model_B
[2022-06-27 13:31:41,972] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_B scheduled__2022-03-08T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:31:42,243] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-03-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-08T00:00:00+00:00
[2022-06-27 13:31:42,245] {python.py:173} INFO - Done. Returned value was: 8
[2022-06-27 13:31:42,362] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_B, execution_date=20220308T000000, start_date=20220627T133141, end_date=20220627T133142
[2022-06-27 13:31:42,489] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:31:42,618] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
